alembic/versions/86504d0cab3f_add_gcn_comment_table.py

```diff
@@ -0,0 +1,107 @@
+"""add GCN comment table
+
+Revision ID: 86504d0cab3f
+Revises: f9611048498a
+Create Date: 2022-01-20 15:25:36.785392
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '86504d0cab3f'
+down_revision = 'f9611048498a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        'comments_on_gcn',
+        sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+        sa.Column('created_at', sa.DateTime(), nullable=False),
```

Comment:
should this default to `now`?

Reply:
I think this is handled in baselayer. I don't see a default applied to any other migrations so I guess it doesn't need to happen in the migration script.